src/main.ts

```diff
@@ -1,28 +1,42 @@
-import {createAppAuth} from '@octokit/auth-app';
-import {Octokit} from '@octokit/rest';
-import * as core from '@actions/core';
+import { createAppAuth } from "@octokit/auth-app";
+import { Octokit } from "@octokit/rest";
+import { Endpoints } from "@octokit/types";
+import * as core from "@actions/core";
 
 async function run(): Promise<void> {
   try {
-    const privateKey: string = core.getInput('private_key');
-    const id: string = core.getInput('app_id');
+    const privateKey: string = core.getInput("private_key");
+    const id: string = core.getInput("app_id");
+    const scope: string = core.getInput("scope");
     const appOctokit = new Octokit({
       authStrategy: createAppAuth,
       auth: {
         id,
         privateKey,
       },
-      baseUrl: process.env['GITHUB_API_URL'] || 'https://api.github.com',
+      baseUrl: process.env.GITHUB_API_URL || "https://api.github.com",
     });
 
-    const {data} = await appOctokit.apps.listInstallations();
+    type listInstallationsResponse = Endpoints["GET /app/installations"]["response"];
```

Comment:
Can you move this type alias out of the function? I think it's a bit confusing to have type aliases mixed inside of runtime code.

Reply:
I moved in https://github.com/getsentry/action-github-app-token/pull/11/commits/2652b2ad37d5d13d6ad7be4d31b4d94e2a35b36a